Identifier.ts

```diff
@@ -35,7 +35,8 @@ export namespace Identifier {
 	export function generate(
 		length: Length,
 		ordering: Extract<Base64.Standard, "ordered" | "reversed">,
-		value: number | Uint8Array | string
+		value: number | Uint8Array | string,
+		bitPrecision?: number
```

Comment:
Think you forgot to do something with this.

Reply:
Thanks!